### DISCUSSION TITLE
IndexOf

### DISCUSSION THREAD

**Discussion by syspone** [balta-io/forum#50] [Tira Dúvidas] [answered]
@andrebaltieri ,

Quando nos deparamos com a seguinte sentença coletada no site da w3shool.

![image](https://user-images.githubusercontent.com/48097255/195478580-6b55e4f7-a815-485e-9a5d-27d71fb53bfc.png)

o retorno é 8, porém não entendi mesmo consultando a documentação o comportamento dos dois parâmetros para indexOf.

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde @cmsoouza, é Java?

De qualquer forma, acredito que esteja usando uma sobrecarga do `indexOf` que além de informar o caractere que deseja obter, informa também a partir de qual índice deseja buscá-lo. No .NET pelo menos é assim.

**Reply by syspone** [reply]
É Java professor, porém estava levando o conceito para c#. Eu só não entendi a posição de onde deve começar contar.
